unskript-ctl/unskript_ctl_notification.py

```diff
@@ -470,10 +467,9 @@ def send_sendgrid_notification(self,
             sg = sendgrid.SendGridAPIClient(api_key)
             sg.send(email_message)
             self.logger.info(f"Notification sent successfully to {to_email}")
-            print(f"Notification sent successfully to {to_email}")
+            self(f"Notification sent successfully to {to_email}")
```

Comment:
remove this

Reply:
This will be tracked via https://unskript.atlassian.net/browse/EN-5240